app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Kindly include `confirmable` on the list of these attributes for the app to work properly with email confirmations.

Reply:
Yeah, I will add this,  but I am not clear about your comment below, please. @elisha2kyakpo1